### DISCUSSION TITLE
Main thread is blocked on Android but not Windows

### DISCUSSION THREAD

**Discussion by danielchalmers** [dotnet/maui#21166] [Q&A]
I've experienced this in a few scenarios, this one is with charts:

https://github.com/apexcharts/Blazor-ApexCharts/assets/7112040/95893ccd-3222-4002-91e6-60302227ec31

https://github.com/apexcharts/Blazor-ApexCharts/assets/7112040/db788c62-8576-4d74-8e53-d4b031965c2b

The code is the same but you can see that animation is blocked on Android while the javascript loads.
Is this a known issue?

**Comment by LitePenguins** [comment]
Having the same issue. Did you fill out a bug report?
